src/utils/getResources.ts

```diff
@@ -0,0 +1,12 @@
+export const getResources = (paths: any) => {
+  const resources: string[] = [];
+
+  Object.keys(paths).forEach((item, index, array) => {
+    if (item.includes("/{id}")) return;
+    if (resources.find(path => path.startsWith("/" + item.split("/")[1])))
```

Comment:
This will return different results depending on what order `/test` and `/test/bar` are encountered.